geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -21,6 +21,16 @@ <h1>{{ document.title }}</h1>
         {% comment %}Translators: Date document was first added to the PGP{% endcomment %}
         <dt class="sr-only">{% translate 'First input' %}</dt>
         <dd>{{ document.log_entries.last.action_time.year }}</dd>
+
+        {% if document.editions %}
+        {% comment %}Translators: Editor label{% endcomment %}
+        <dt>{% translate 'Editor' %}</dt>  {# optionally pluralize? #}
+        {% for ed in document.editions %}
+        <dd>Ed. {{ ed.source }}{% if ed.notes %}. {{ ed.notes }}{% endif %}</dd>
```

Comment:
this "Ed." seems redundant with the label (and harder to translate, maybe)

Reply:
good point! I was following their current conventions, but I agree that doesn't make sense here